radixdlt-java-common/src/main/java/com/radixdlt/networks/Network.java

```diff
@@ -61,12 +59,8 @@ public int getId() {
 		return id;
 	}
 
-	public Optional<Txn> genesisTxn() {
```

Comment:
Like mentioned above this should remain here. Perhaps just change it to a String.

Reply:
Fixed